src/planscape/config/attributes.json

```diff
@@ -0,0 +1,41 @@
+{
+    "regions": [
+        {
+            "region_name": "sierra_cascade_inyo",
+            "filepath": "data/sierra_nevada",
+            "display_name": "Sierra Nevada",
+            "attributes": [
+                {
+                    "attribute_name": "distance_from_road",
+                    "filepath": "attributes",
+                    "filename": "distance_from_roads_300m.tif",
+                    "display_name": "Distance From Road",
+                    "source": "",
+                    "source_link": "",
+                    "data_provider": "",
+                    "data_download_link": "",
+                    "data_year": "",
+                    "reference_link": "",
+                    "min_value": "",
+                    "max_value": "",
```

Comment:
@JanLauGe 
@elsiel23 

Might you have suggestions for populating these source fields?
1) are all the fields necessary or just a subset? (I copied them from config/conditions.json)
2) what are their values? I'm not as familiar with the data sources as you two.
3) is there additional info I can link to? (e.g. Laurens's scripts for computing distance-from-roads)
4) I'd like to have these added in a follow-up PR (to unblock follow-up tasks). Does anyone want to take over populating these fields?

Reply:
is the goal to use the same metadata fields as the other raster files? i think perhaps a subset might be enough for now. a couple thoughts:

- since we're deriving some of the data, data_download_link might not be available for several rasters
- same with data_provider 
- we could link our documentation on how we develop the layers for reference_link 
- min_value and max_value are probably still useful

Reply:
Can add a todo for a later contributor to add suggestions.

Reply:
thanks! I've added a subset of the data fields.